### ISSUE TITLE
[1.18] Item glow rendering over EMI

### ISSUE THREAD

**Issue by slava110** [RealRTTV/malum-quilt#15]
![2022-07-02_19 27 24](https://user-images.githubusercontent.com/26705162/177213189-99d926f0-c875-4dbe-b36c-5efbfbd10e78.png)
I know that this is intended that glow effect is made like particles but I think it shouldn't render if there's anything over it. Otherwise it looks meh. What do you think?

**Comment by Arathain** [comment]
got it working + got instanced particle stacks to fully work, will be included in next release

**Comment by slava110** [comment]
Nice! Idk what instanced particle stacks are but... Will it fix #12 too maybe? Would be cool, huh

**Comment by Arathain** [comment]
no, that's an intended feature - instanced particle emitters mean that the same item is able to have its particles rendered multiple times.
